code_challenges.rb

```diff
@@ -20,10 +23,22 @@
 letter_a = 'a'
 # Expected output: ['tea', 'soda water']
 
-
-
-
-
+# use include?
+result = []
+beverages_array.each do | beverages_array |
+  if beverages_array.include? 'a'
+    result = beverages_array
+    p [beverages_array]
+  end
+end
+
+result = []
+beverages_array.each do | beverages_array |
+  if beverages_array.include? 'o'
+    result = beverages_array
+    p [beverages_array]
+  end
+end
```

Comment:
ok cool! we have some blocks that are working for us and that's great. but this logic is not wrapped in a method so isn't very reusable. how can we make this into a method that takes an array and a letter as an argument? this code only works with 'a' or 'o'. but what about other letters?